waiter/src/waiter/core.clj

```diff
@@ -163,6 +163,13 @@
                   (cid/ensure-correlation-id nested-response get-request-cid)
                   nested-response)))))))))
 
+(defn wrap-support-info
```

Comment:
I do not like having to wrap `support-info` on every request. Should we instead have a middleware that catches any thrown exception and renders the response? That way there will be no need to have to pass the request around.